assets/src/scripts/components/hydrograph/scales.js

```diff
@@ -14,9 +15,20 @@ const paddingRatio = 0.2;
  */
 function extendDomain(domainExtent) {
     const padding = paddingRatio * (domainExtent[1] - domainExtent[0]);
+
+    // If all values are above zero, make zero the lower bound
+    if (domainExtent[0] >= 0 && domainExtent[1] >= 0) {
+        return [
+            Math.max(0, domainExtent[0] - padding),
```

Comment:
Do we have a preference for using Math.max over d3.max? I have been using d3.max in the tooltip code for basically the same purpose as here where I'm finding the max of two numbers. I used d3.max because I was already using it in the package to find the maximum of an array.

Reply:
I personally don't have a preference, though I've been using `Math` methods recently (e.g. `Math.abs`), but that's because I couldn't find a d3 absolute value method.

Reply:
I wasn't aware of d3.max - curious if it does something different than the standard one, which I would assume would be pretty optimized.

Reply:
In `d3-scale, only `Math.max` is used... From the docs, I assume there's specific cases where `d3.max` is useful but I don't think we need it here.